packages/squiggle-lang/__tests__/Distributions/AlgebraicCombination_test.res

```diff
@@ -0,0 +1,313 @@
+open Jest
+open Expect
+open TestHelpers
+
+let {
+  normalDist5,  // mean=5, stdev=2
+  normalDist10,  // mean=10, stdev=2
+  normalDist20,  // mean=20, stdev=2
+  normalDist,  // mean=5; stdev=2
+  uniformDist,  // low=9; high=10
+  betaDist,  // alpha=2; beta=5
+  lognormalDist,  // mu=0; sigma=1
+  cauchyDist,  // local=1; scale=1
+  triangularDist,  // low=1; medium=2; high=3;
+  exponentialDist,  // rate=2
+} = module(GenericDist_Fixtures)
+
+let {
+    algebraicAdd, 
+    algebraicMultiply, 
+    algebraicDivide, 
+    algebraicSubtract, 
+    algebraicLogarithm, 
+    algebraicPower
+} = module(DistributionOperation.Constructors)
+
+let algebraicAdd = algebraicAdd(~env)
+let algebraicMultiply = algebraicMultiply(~env)
+let algebraicDivide = algebraicDivide(~env)
+let algebraicSubtract = algebraicSubtract(~env)
+let algebraicLogarithm = algebraicLogarithm(~env)
+let algebraicPower = algebraicPower(~env)
+
+describe("(Algebraic) addition of distributions", () => {
+
+    describe("mean", () => {
+        test("normal(mean=5) + normal(mean=20)", () => {
```

Comment:
I feel like the following code could be refactored a bit to be nicer. Like,

```rescript
test("mean(normal(mean=5) + normal(mean=20)) = 25", () => {
  let mean = getMean(algebraicAdd(normalDist5, normalDist2));
  let expectedResult = Some(2.5e1)
  expect(mean).toBe(expectedResult)
``

We could even abstract it further, because there could be a bunch of these tests. Something like,
```rescript
let meanExpectations = [
  (normalDist5, normalDist2, 2.5e1),
  (uniformDist, betaDist, 0.01927225696028752),
   ...
]
```